### DISCUSSION TITLE
Parse Site information from IGC file

### DISCUSSION THREAD

**Discussion by lud-hu** [Turbo87/igc-parser#217] [Ideas]
Hi,

currently the igc-parser is not parsing the information about the flying site from the igc file.
E.g.:
```
HOSITSite:Bassano
```
According to the Specification it describes: 
```
SIT H Site, Name, region, nation etc
```
http://www.ukiws.demon.co.uk/GFAC/documents/tech_spec_gnss.pdf

Are there plans to implement parsing of this?

Thanks for the great module! :)

**Comment by Turbo87** [comment]
tbh I have never seen this header being used anywhere before 😅 

feel free to work on a pull request, but it seems like a rarely used feature to me 🤔

**Comment by lud-hu** [comment]
The XCTrack App is exporting this. :D
Thanks, I will see if I can do it!
